packages/protocol/scripts/bash/console.sh

```diff
@@ -26,4 +26,4 @@ if ! nc -z 127.0.0.1 8545 && [ -z $FORNO ] ; then
   exit 1
 fi
 
-yarn run truffle console --network $NETWORK --build_directory $PWD/build/$NETWORK $FORNO
+yarn run truffle console --network $NETWORK --contracts_build_directory $PWD/build/contracts $FORNO
```

Comment:
this is surprising to me as I think we use this `build_directory` elsewhere
we have been using `release-lib.sh` to populate `build/$TAG`

Reply:
Yeah I also saw `build_directory` used in different places, but after consulting docs and some source code this is the only way I could get the artefacts to automatically show up as truffle contracts in the global scope in the console. 🤷